package/scripts/install-npm.js

```diff
@@ -3,13 +3,20 @@ const path = require("path");
 
 console.log("Updating symlinks for Android build...");
 
+const isExists = (path) => {
+  return fs.existsSync(path);
+}
```

Comment:
Not obligatory, but prettier want a semicolon 

```suggestion
};
```